test-fixtures/test-application-ts/src/index.ts

```diff
@@ -226,3 +227,23 @@ function testWait(client: showcase.v1beta1.EchoClient) {
     assert.deepStrictEqual(response.content, request.success.content);
   });
 }
+
+function testWaitDecodeLRO(client: showcase.v1beta1.EchoClient) {
+  it('waitDecodeLRO', async function() {
+    this.timeout(10000);
+    const request = {
+      ttl: {
+        seconds: 5,
+        nanos: 0,
+      },
+      success: {
+        content: 'done',
+      },
+    };
+    const [operation] = await client.wait(request);
+    const decodedOperation = await client.waitDecodeLRO(operation.name!);
+    assert.deepStrictEqual(decodedOperation.name, operation.name);
+    assert(decodedOperation.metadata);
+    assert(decodedOperation.result)
```

Comment:
We can actually check it to be the expected result (look at `response.content`). That way we'll also verify that the type is correct. Look at the test for `wait`.

(the same applies to the JavaScript test application)

Reply:
sure, added the new checks for the content from response in test application. Thanks!